aten/src/ATen/native/native_functions.yaml

```diff
@@ -2488,19 +2488,23 @@
 
 - func: quantize_linear(Tensor self, float scale, int zero_point) -> Tensor
   variants: function, method
-  requires_tensor: True
+  dispatch:
+    CPU: quantize_linear_cpu
 
 - func: dequantize(Tensor self) -> Tensor
   variants: function, method
-  requires_tensor: True
+  dispatch:
+    QuantizedCPU: dequantize_quant
 
 - func: q_scale(Tensor self) -> Scalar
   variants: function, method
-  requires_tensor: True
+  dispatch:
+    QuantizedCPU: q_scale_quant
```

Comment:
should we even dispatch this and q_zero_point as QuantizedCPU?  I guess it's fine, but you need to redispatch anyway.  Either way is fine for now I guess.

Reply:
you mean guarding against other schemes in q_scale_quant function? 
this is can guard other tensor types at least, we can guard other qschemes in the function later.